src/models/payments.js

```diff
@@ -25,6 +25,10 @@ function * settle (sourceTransfer, destinationTransfer, config, ledgers) {
           code: 'S00',
           name: 'Bad Request',
           message: 'destination transfer failed: ' + err.message
+        } : (err.name === 'NotAcceptedError') ? {
```

Comment:
If the plugin returns a `NotAcceptedError`, is that always due to `Insufficient Liquidity`? This is what RFC-0004 says about `NotAcceptedError`:
> An operation has been rejected due to ledger-side logic

I guess a `NotAcceptedError` can not always be translated into `Insufficient Liquidity`.